README.md

```diff
@@ -1,104 +1,157 @@
-# pravega-samples
+# Pravega and Analytics Connectors Examples
 
-Sample applications for Pravega.
+This repository contains code samples to demonstrate how developers can work with 
```

Comment:
I think we also need to mention about the `master` and `develop` branch. 
- `master` will point to release artifacts of pravega and connector
- `develop` is the development branch which will point to snapshot artifacts of pravega and connector

Reply:
Thanks, I mentioned these branches in the instructions to build `pravega-samples`.